### ISSUE TITLE
In Ubuntu 14.04.2 run Error

### ISSUE THREAD

**Issue by HotWordland** [vapor/jwt#13]
I used vapor-jwt run heroku have no problem. but I used in Ubuntu 14.04.2 run error.
information:
execute this line.
`
        let jwt = try JWT(payload: Node(ExpirationTimeClaim(Date() + 60)),
                          signer: HS256(key: "secret"))
`

*** Error in `.build/debug/App': free(): invalid next size (fast): 0x00007f3544013fe0 ***

**Comment by siemensikkema** [comment]
@HotWordland I don't have access to Ubuntu currently. Could you please try the following to see if it is specific to HMAC signers?
```
let jwt = try JWT(payload: Node(ExpirationTimeClaim(Date() + 60)), signer: Unsigned())
```
and 
```
let jwt = try JWT(payload: Node(ExpirationTimeClaim(Date() + 60)), signer: ES256(encodedKey: "AL3BRa7llckPgUw3Si2KCy1kRUZJ/pxJ29nlr86xlm0=")
```
Thanks!

**Comment by tannernelson** [comment]
This seems like an issue with https://github.com/vapor/clibressl.

**Comment by HotWordland** [comment]
@siemensikkema Thanks.
I fix this problem in Ubuntu 14.04.2 with line:
```
let jwt = try JWT(payload: Node(ExpirationTimeClaim(Date() + 60)), signer: Unsigned())
```

**Comment by vzsg** [comment]
Don't do that for the real application! Using unsigned tokens is an enormous security hole, but it was necessary to see where the error is coming from.

**Comment by vzsg** [comment]
I tried to reproduce the issue using Docker. All the images I found were built on Ubuntu 14.04.5 LTS, so I picked `vapor/vapor:1.0-trusty`, set up the test project, and... it doesn't show these symptoms, signing with HS256 works cleanly.

Which brings me to some questions:
- What is the output of `uname -a`?
- What is the output of `swift -version` and how did you install it?
- What is the current `clang` version?
